src/main/java/pl/javastart/task/ArrayUtils.java

```diff
@@ -0,0 +1,26 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class ArrayUtils {
+    Scanner scanner = new Scanner(System.in);
+
+    double[] createArray() {
+        System.out.println("Podaj rozmiar tablicy: ");
+        int number = scanner.nextInt();
+        double[] array = new double[number];
+        System.out.printf("Wprowadź %d liczb:\n", number);
+        for (int i = 0; i < array.length; i++) {
+            array[i] = scanner.nextDouble();
+        }
+        return array;
+    }
+
+    void calculateSum(double[] array) {
```

Comment:
Preferuj definiowanie metod, które zwracają wynik, a wyświetlaniem zajmuj się w osobnej klasie (np. w metodzie main). 
```
double calculateSum(double[] array) { ... }
```
Taki kod jest bardziej uniwersalny, ponieważ jeżeli metoda zwróci wynik, to dopiero później decydujemy, czy chcemy go wyświetlić w konsoli, na stronie internetowej, czy może w jakimś okienku